src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -140,7 +140,7 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
                     {
                         default:
                         case ImportMode.Auto:
-                            copyOnly = downloadClientItem != null && !downloadClientItem.CanMoveFiles;
```

Comment:
Instead of splitting the logic here and in the download client, `CanMoveFiles` should not be set to true if remove is false

Reply:
Actually, we already do a similar check in `DownloadEventHub`, no point tackling changing this, maybe something to consider if we refactor the import process.